packages/plugin-default-event-tracking-advanced-browser/README.md

```diff
@@ -65,7 +65,7 @@ Examples:
 
 |Name|Type|Default|Description|
 |-|-|-|-|
-|`cssSelectorAllowlist`|`string[]`|`undefined`| When provided, only allow elements matching any selector to be tracked. |
+|`cssSelectorAllowlist`|`string[]`|`['a', 'button', 'input', 'select', 'textarea', 'label']`| When provided, only allow elements matching any selector to be tracked. |
```

Comment:
since we are supporting tags, should this be better named as querySelectorAllowlist or something?

Reply:
I prefer css selector more as query selector is a method name for querying elements with CSS selectors, e.g., https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector#selectors:
> A string containing one or more selectors to match. This string must be a valid CSS selector string; if it isn't, a SyntaxError exception is thrown.

It mentions the [CSS selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors) concept, which includes a list of all [selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors#selectors), like id, class, type (tag).

Reply:
ahh i see i see